### PR TITLE
Change Uyuni to Leap 15.3

### PR DESCRIPTION
**Pull request by juliogonzalez**
## What does this PR change?

Next Uyuni version is going to be Leap 15.3

Change Server and Proxy to openSUSE Leap 15.3, as well as the examples.

WARNING: ci-pr image is not ready yet, @jordimassaguerpla is having a look (https://build.opensuse.org/package/show/systemsmanagement:sumaform:images:libvirt/opensuse153-ci-pr)

### CODE REVIEW

**Review by Bischoff — APPROVED** [review]
LGTM

File: backend_modules/libvirt/base/main.tf [hunk 1]
```diff
@@ -12,6 +12,8 @@ locals {
     opensuse151o = "${var.use_mirror_images ? "http://${var.mirror}" : "https://download.opensuse.org"}/distribution/leap/15.1/jeos/openSUSE-Leap-15.1-JeOS.x86_64-15.1.0-OpenStack-Cloud-Current.qcow2"
     opensuse152-ci-pr  = "${var.use_mirror_images ? "http://${var.mirror}" : "https://download.opensuse.org"}/repositories/systemsmanagement:/sumaform:/images:/libvirt/images/opensuse152-ci-pr.x86_64.qcow2"
     opensuse152o = "${var.use_mirror_images ? "http://${var.mirror}" : "https://download.opensuse.org"}/distribution/leap/15.2/appliances/openSUSE-Leap-15.2-JeOS.x86_64-OpenStack-Cloud.qcow2"
+    opensuse153-ci-pr  = "${var.use_mirror_images ? "http://${var.mirror}" : "https://download.opensuse.org"}/repositories/systemsmanagement:/sumaform:/images:/libvirt/images/opensuse153-ci-pr.x86_64.qcow2"
```
Comment (srbarrios) [inline backend_modules/libvirt/base/main.tf:15]: This image it doesn't seems to exist yet. 
@jordimassaguerpla maybe you know how to fix it?

File: modules/base/variables.tf [hunk 2]
```diff
@@ -65,6 +65,6 @@ variable "provider_settings" {
 
 variable "images" {
   description = "list of images to be uploaded to the libvirt host, leave default for all"
-  default     = ["centos6o", "centos7o", "centos8o", "opensuse150o", "opensuse151o", "opensuse152o", "sles11sp4", "sles12", "sles12sp1", "sles12sp2", "sles12sp3", "sles12sp4o", "sles12sp5o", "sles15o", "sles15sp1o", "sles15sp2o", "sles15sp3o", "ubuntu1604o", "ubuntu1804o", "ubuntu2004o"]
+  default     = ["centos6o", "centos7o", "centos8o", "opensuse150o", "opensuse151o", "opensuse152o", "openmsuse153o", "sles11sp4", "sles12", "sles12sp1", "sles12sp2", "sles12sp3", "sles12sp4o", "sles12sp5o", "sles15o", "sles15sp1o", "sles15sp2o", "sles15sp3o", "ubuntu1604o", "ubuntu1804o", "ubuntu2004o"]
```
Comment (srbarrios) [inline modules/base/variables.tf:68]: typo? open*m*suse153o

File: modules/cucumber_testsuite/variables.tf [hunk 3]
```diff
@@ -31,7 +31,7 @@ variable "name_prefix" {
 
 variable "images" {
   description = "list of images to be uploaded to the libvirt host, leave default for all"
-  default     = ["centos6o", "centos7o", "centos8o", "opensuse150o", "opensuse151o", "opensuse152o", "sles11sp4", "sles12", "sles12sp1", "sles12sp2", "sles12sp3", "sles12sp4o", "sles12sp5o", "sles15o", "sles15sp1o", "sles15sp2o", "sles15sp3o", "ubuntu1604o", "ubuntu1804o", "ubuntu2004o"]
+  default     = ["centos6o", "centos7o", "centos8o", "opensuse150o", "opensuse151o", "opensuse152o", "openmsuse153o", "sles11sp4", "sles12", "sles12sp1", "sles12sp2", "sles12sp3", "sles12sp4o", "sles12sp5o", "sles15o", "sles15sp1o", "sles15sp2o", "sles15sp3o", "ubuntu1604o", "ubuntu1804o", "ubuntu2004o"]
```
Comment (srbarrios) [inline modules/cucumber_testsuite/variables.tf:34]: typo? open**m**suse153o

File: main.tf.aws-testsuite.example [hunk 4]
```diff
@@ -20,7 +20,7 @@ module "cucumber_testsuite" {
   cc_username = ...
   cc_password = ...
 
-  images = ["opensuse150", "opensuse151"]
+  images = ["opensuse152", "opensuse153"]
```
Comment (srbarrios) [inline main.tf.aws-testsuite.example:23]: I don't see a good reason to download "opensuse152"  image

File: backend_modules/libvirt/base/main.tf (hunk 1 above)
Comment (juliogonzalez) [reply backend_modules/libvirt/base/main.tf:15]: @srbarrios looks to me you didn't read the PR description :-)

File: main.tf.aws-testsuite.example (hunk 4 above)
Comment (juliogonzalez) [reply main.tf.aws-testsuite.example:23]: The controller is still SP2.

File: modules/base/variables.tf (hunk 2 above)
Comment (juliogonzalez) [reply modules/base/variables.tf:68]: Right, I will fix it.

File: modules/cucumber_testsuite/variables.tf (hunk 3 above)
Comment (juliogonzalez) [reply modules/cucumber_testsuite/variables.tf:34]: Yes, it's a typo. Will fix it.

File: backend_modules/aws/base/main.tf [hunk 5]
```diff
@@ -82,6 +82,27 @@ data "aws_ami" "opensuse152" {
   }
 }
 
+data "aws_ami" "opensuse153" {
```
Comment (rjmateus) [inline backend_modules/aws/base/main.tf:85]: you need to add this image to the ami info in the configuration output at: https://github.com/uyuni-project/sumaform/blob/master/backend_modules/aws/base/main.tf#L413

File: backend_modules/azure/base/main.tf [hunk 6]
```diff
@@ -24,6 +24,13 @@ data "azurerm_platform_image" "opensuse152" {
   sku       = "15-2"
 }
 
+data "azurerm_platform_image" "opensuse153" {
```
Comment (rjmateus) [inline backend_modules/azure/base/main.tf:27]: this image information needs to be added to the module output: https://github.com/uyuni-project/sumaform/blob/master/backend_modules/azure/base/main.tf#L74

**Review by rjmateus — COMMENTED** [review]
Just two comments to review.
Thanks for the contribution

File: backend_modules/aws/base/main.tf (hunk 5 above)
Comment (juliogonzalez) [reply backend_modules/aws/base/main.tf:85]: Done.

File: backend_modules/azure/base/main.tf (hunk 6 above)
Comment (juliogonzalez) [reply backend_modules/azure/base/main.tf:27]: Done

File: backend_modules/libvirt/base/main.tf (hunk 1 above)
Comment (juliogonzalez) [reply backend_modules/libvirt/base/main.tf:15]: Image is created now.

**Pull request merged**